Assets/Packages/com.studio23.ss2.roomloadingsystem/Samples/RoomLoadingSystemDemo1/Scripts/PlayerDetector.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Studio23.SS2.RoomLoadingSystem.Core;
+using UnityEngine;
+
+namespace Studio23.SS2.RoomLoadingSystem.Samples.Demo1
+{
+    /// <summary>
+    /// exmaple class for setting room manager player ref
+    /// </summary>
+    public class PlayerDetector:MonoBehaviour
```

Comment:
Maybe the player game object will have a singleton identifier script that will allow any script to get a reference to the player Transform whenever needed?

Reply:
The room loader just needs a reference to the player transform. This is just a sample showing how that can be set.
Singleton identifier is a good idea but  I don't want to enforce that.